apps/st2-history/history-panel.component.js

```diff
@@ -79,7 +79,7 @@ export default class HistoryPanel extends React.Component {
           store.dispatch({ type: 'SELECT_EXECUTION' });
         }
       })
-      ;
+    ;
```

Comment:
I usually indent `;` too, but I'm open to doing it this way too so long as there is a eslint rule to remind me.